src/parameters.toml

```diff
@@ -1179,6 +1179,23 @@ value = -10.66873
 type = "float"
 description = "Coefficient, c, for calculating immersion freezing J_het of Illite [-]. From Knopf and Alpert 2013. DOI: 10.1039/C3FD00035D."
 
+# Microphysics - P3 ice nucleation paramerizations
+
+[Thompson2004_T_threshold_Cooper]
+value = 233
```

Comment:
We actually already have this temperature also. It's here: https://github.com/CliMA/CLIMAParameters.jl/blob/1f3cf67c4296a2f780e55e85660c6e8cd0395c72/src/parameters.toml#L1651